.github/workflows/approved-for-ci-run.yml

```diff
@@ -69,16 +69,43 @@ jobs:
         with:
           ref: main
           token: ${{ secrets.CI_ACCESS_TOKEN }}
+      
+      - name: Look for existant PR
+        id: "get_pr"
+        run: |
+          ALREADY_CREATED="$(gh pr --repo ${GITHUB_REPOSITORY} list --head ${BRANCH} --base main --json number --jq '.[].number')"
+          echo "ALREADY_CREATED=$ALREADY_CREATED" >> $GITHUB_OUTPUT
+      
+      - name: Get changed labels
+        id: "get_labels"
+        env:
+          ALREADY_CREATED: ${{ steps.get_pr.outputs.ALREADY_CREATED }}
+        run: |
+          LABELS_TO_ADD=
+          LABELS_TO_REMOVE=
+          if [ -n "$ALREADY_CREATED" ]; then
+            LABELS_TO_REMOVE=$(comm -23 <(gh pr --repo ${GITHUB_REPOSITORY} view $ALREADY_CREATED --json labels --jq '.labels.[].name'| grep -E '^run' | sort) <(gh pr --repo ${GITHUB_REPOSITORY} view $PR_NUMBER --json labels --jq '.labels.[].name' |  grep -E '^run' | sort ))
+            LABELS_TO_REMOVE=${LABELS_TO_REMOVE//run-e2e-tests-in-draft/}
+            LABELS_TO_REMOVE=$(echo $LABELS_TO_REMOVE | awk 'BEGIN {list=""} {for (i=1;i<=NF;i++) if (length($i)>0) if (length(list)>0) list=list "," $i; else list=$i;} END {print list}')
```

Comment:
Is this logic to join items using `,`?
Can we use [`paste(1)`](https://linux.die.net/man/1/paste)? Something like `$(echo $LABELS_TO_REMOVE | paste -sd ',' -)`

Reply:
done